### PR TITLE
Release 1 : New Layout UI

### PR DESCRIPTION
**Pull request by theaubmov**
## Summary by CodeRabbit

- **New Features**
  - Implemented a new layout for the properties panel.
  - Enhanced zoom functionality after importing BPMN models.
  - Introduced new tabs and buttons in the UI for improved navigation and modeler interaction.
  - Added new CSS file for extended styling options.
  - Updated the properties panel to automatically prioritize certain groups.

- **Style Updates**
  - Refined the appearance of the search bar, buttons, and other UI components for a more cohesive look.
  - Adjusted the layout and styling of the BPMN palette for better usability.
  - Overhauled CSS to improve readability and maintainability.

- **Bug Fixes**
  - Corrected titles for Data Input and Data Output in the BPMN modeler.
  
- **Documentation**
  - Updated charset meta tag in the HTML for better standards compliance.

- **Refactor**
  - Streamlined event listeners and tab setup in the BPMN modeler for efficiency.
  - Simplified logic in extension helper functions for clarity.

- **Chores**
  - Reorganized import statements and aligned code for better project structure.

### CODE REVIEW

File: app/spiffworkflow/InputOutput/IoPalette.js [hunk 1]
```diff
@@ -37,26 +52,268 @@ IoPalette.prototype.getPaletteEntries = function() {
...
+    'create.user-task': {
+      group: 'activities',
+      className: 'bpmn-icon-user',
+      title: translate('User Task'),
+      action: {
+        dragstart: createShape('bpmn:UserTask'),
+        click: createShape('bpmn:UserTask')
+      }
+    },
+    'create.scirpt-task': {
```
Comment (burnettk) [inline app/spiffworkflow/InputOutput/IoPalette.js:110]: typo

**Review by burnettk — APPROVED** [review]
i thought i saw one time in the slack movie you posted when you said "Update : I ve completed properties panel customization, as shown in the screenshot, we have now the ability to highlight a specifc group and mark it as default one.  so it ll be displayed in the default tab." that the active properties panel "thing" did not have an arrow pointing down (you know the one that shows that it is currently "open").

image

**Comment by jbirddog** [comment]
@coderabbitai review

**Comment by theaubmov** [comment]
@burnettk , the issue of active properties panel is resolved.

**Pull request merged**